src/overlays/actors/ovl_En_Test6/z_en_test6.c

```diff
@@ -848,166 +923,170 @@ void EnTest6_Update(Actor* thisx, PlayState* play) {
-                    this->unk_154 = 100.0f;
+                    this->clockSpeed = 100.0f;
                 }
 
                 if (play->csCtx.actorCues[cueChannel]->startPos.y != 0) {
-                    this->unk_14C = (u32)play->csCtx.actorCues[cueChannel]->startPos.y;
+                    this->speed = (u32)play->csCtx.actorCues[cueChannel]->startPos.y;
                 } else {
-                    this->unk_14C = 20.0f;
+                    this->speed = 20.0f;
                 }
 
                 if (play->csCtx.actorCues[cueChannel]->startPos.z != 0) {
-                    this->unk_150 = (u32)play->csCtx.actorCues[cueChannel]->startPos.z;
+                    this->clockDist = (u32)play->csCtx.actorCues[cueChannel]->startPos.z;
                 } else {
-                    this->unk_150 = 300.0f;
+                    this->clockDist = 300.0f;
                 }
-                this->unk_158 = 0.0f;
+                this->clockAccel = 0.0f;
                 break;
 
-            case 7:
+            case SOTCS_CUEID_RESET_CLOCKS_SLOW_DOWN:
                 if (play->csCtx.actorCues[cueChannel]->startPos.x != 0) {
-                    this->unk_158 = (u32)play->csCtx.actorCues[cueChannel]->startPos.x;
+                    this->clockAccel = (u32)play->csCtx.actorCues[cueChannel]->startPos.x;
                 } else {
-                    this->unk_158 = -5.0f;
+                    this->clockAccel = -5.0f;
                 }
-                this->unk_154 += this->unk_158;
+                this->clockSpeed += this->clockAccel;
                 break;
 
-            case 8:
+            case SOTCS_CUEID_RESET_CLOCKS_SPEED_UP:
                 if (play->csCtx.actorCues[cueChannel]->startPos.x != 0) {
-                    this->unk_158 += (u32)play->csCtx.actorCues[cueChannel]->startPos.x;
+                    this->clockAccel += (u32)play->csCtx.actorCues[cueChannel]->startPos.x;
                 } else {
-                    this->unk_158 += 2.0f;
+                    this->clockAccel += 2.0f;
                 }
 
-                this->unk_154 += this->unk_158;
-                if (this->unk_154 > 10000.0f) {
-                    this->unk_154 = 10000.0f;
-                    this->cueId = 0;
+                this->clockSpeed += this->clockAccel;
+                if (this->clockSpeed > 10000.0f) {
+                    this->clockSpeed = 10000.0f;
+                    this->cueId = SOTCS_CUEID_NONE;
                 }
                 break;
 
-            case 0:
+            case SOTCS_CUEID_NONE:
             default:
-                this->unk_276 = 99;
+                this->drawType = SOTCS_DRAW_TYPE_NONE;
                 return;
 
-            case 9:
+            case SOTCS_CUEID_DOUBLE_END:
                 Play_SetRespawnData(&play->state, 1, ((void)0, gSaveContext.save.entrance & 0xFFFF), player->unk_3CE,
                                     PLAYER_PARAMS(0xFF, PLAYER_INITMODE_B), &player->unk_3C0, player->unk_3CC);
```

Comment:
```suggestion
                Play_SetRespawnData(&play->state, RESPAWN_MODE_RETURN, ((void)0, gSaveContext.save.entrance), player->unk_3CE,
                                    PLAYER_PARAMS(0xFF, PLAYER_INITMODE_B), &player->unk_3C0, player->unk_3CC);
```
Locally the `& 0xFFFF` doesn't seem to be needed, and can use the `RESPAWN_MODE` enum.